include/graphblas/reference/blas2.hpp

```diff
@@ -2824,6 +2825,7 @@ namespace grb {
 	 * @see grb::eWiseLambda for the user-level specification.
 	 */
 	template<
+		Descriptor descr = descriptors::no_operation,
```

Comment:
(it seems forgotten here as well -- note that the dense descriptor is only defined to apply for vectors (x), and not for A)